codalab/lib/zip_util.py

```diff
@@ -47,11 +48,14 @@ def strip_archive_ext(path):
     raise UsageError('Not an archive: %s' % path)
 
 
-def unpack(ext, source, dest_path):
-    """
-    Unpack the archive |source| to |dest_path|.
-    Note: |source| can be a file handle or a path.
-    |ext| contains the extension of the archive.
+def unpack(ext: str, source: Union[str, IO[bytes]], dest_path):
```

Comment:
should we have a type annotation for `dest_path`?

```suggestion
def unpack(ext: str, source: Union[str, IO[bytes]], dest_path: str):
```